packages/scope-manager/package.json

```diff
@@ -2,20 +2,36 @@
   "name": "@typescript-eslint/scope-manager",
   "version": "5.51.0",
   "description": "TypeScript scope analyser for ESLint",
-  "keywords": [
-    "eslint",
-    "typescript",
-    "estree"
-  ],
-  "engines": {
-    "node": "^14.18.0 || ^16.0.0 || >=18.0.0"
-  },
   "files": [
     "dist",
     "package.json",
     "README.md",
     "LICENSE"
   ],
+  "type": "commonjs",
+  "typesVersions": {
+    "<3.8": {
+      ".": [
+        "_ts3.4/dist/index.d.ts"
+      ]
+    }
```

Comment:
TypeScript 3.8 came out 3 years ago. I'd vote to just get rid of <3.8 compat while we're in the area & doing breaking changes.

Reply:
#5915 - our minimum is now 4.2 - so yeah we can bump this up.
I'll do that separately